src/domain/event/EventSummaryPage.tsx

```diff
@@ -189,7 +193,7 @@ const EventSummaryPage: React.FC = () => {
                       })}
                     </span>
                   </h2>
-                  {!isEventDraft && (
+                  {!isEventDraft && enrolmentType === EnrolmentType.Internal && (
```

Comment:
minor but result of `enrolmentType === EnrolmentType.Internal` could be saved to a variable just like isEventDraft to make this little bit more concise and easier to read